executable/dex.ksy

```diff
@@ -5,6 +5,8 @@ meta:
   license: CC-BY-SA-3.0
   # Portions of this page are reproduced from work created and shared by Google and used
   # according to terms described in the Creative Commons 3.0 Attribution License.
+  imports:
+    - ../common/vlq_base128_le
```

Comment:
I guess it should be `/common/vlq_base128_le`, as this is considered a "standard library" type. It shouldn't depend on where the dex.ksy is located.

Reply:
Makes sense. I'll strip the leading periods.